File: src/test/java/de/codecentric/springnewspaper/NewspaperStubs.java
package de.codecentric.springnewspaper;

import de.codecentric.springnewspaper.configuration.JacksonConfiguration;
import de.codecentric.springnewspaper.domain.Newspaper;

import java.io.IOException;

class NewspaperStubs {
    public static Newspaper getNewspaper1() throws IOException {
        return JacksonConfiguration.applicationMapper().readValue(
                NewspaperStubs.class.getResource("/softwerker.json"), Newspaper.class
        );
    }
}

File: src/test/kotlin/de/codecentric/springnewspaper/NewspaperStubs.kt
package de.codecentric.springnewspaper

import de.codecentric.springnewspaper.configuration.applicationMapper
import de.codecentric.springnewspaper.domain.Newspaper

object NewspaperStubs {
    val newspaper1 by lazy {
        applicationMapper().readValue(this.javaClass.getResource("/softwerker.json"), Newspaper::class.java)
    }
}